source/addonHandler/__init__.py

```diff
@@ -337,15 +337,12 @@ def completeRemove(self,runUninstallTask=True):
 
 	def addToPackagePath(self, package):
 		""" Adds this L{Addon} extensions to the specific package path if those exist.
-		This allows the addon to run.
+		This allows the addon to "run" / be available via `globalPluginHandler.runningPlugins`.

```

Comment:
I think it's the same mechanism for braille drivers?

In `braille.initialize()` there is a call to `config.addConfigDirsToPythonPackagePath(brailleDisplayDrivers)` which, for each addon in `addonHandler.getRunningAddons()` calls `addonHandler.addToPackagePath()`. This updates the `brailleDisplayDrivers__path__`

Perhaps I should extend this comment to also mention all the modules that call `config.addConfigDirsToPythonPackagePath`: 
- `globalPlugins`
- `appModules`
- `synthDrivers`
- `brailleDisplayDrivers`


Perhaps something like:

```suggestion
		This allows the addon to "run" / be available because the package is able to search its path,
		looking for particular modules. This is used by the following:
		- `globalPlugins`
		- `appModules`
		- `synthDrivers`
		- `brailleDisplayDrivers`
```

Reply:
Yes, this exactly describes what it does now.